test/config/fileserver_config.json

```diff
@@ -0,0 +1,41 @@
+{"os_image":

```

Comment:
Cool!  This file still needs to go on the internal EOS2 repo, dellemc-test, as it has lab specific data in it.    You then can specify the config dir and extra file on the command line.  example:  run_tests.py --config config-mn --extra fileserver_config.json -test xxxx.  Ping me to help get this set up with you.